tests/support/views/api-rules-view.js

```diff
@@ -36,3 +36,25 @@ Cypress.Commands.add('createApiRule', (ApiRuleName, ApiRuleHost) => {
     .contains('Discovery and Network')
     .click();
 });
+
+Cypress.Commands.add('checkApiRuleStatus', ApiRuleName => {
```

Comment:
Shouldn't we move it to support/commands file?

Reply:
it's connected to api rules view so I think it can stay here